components/brave_wallet_ui/components/shared/password-input/style.ts

```diff
@@ -0,0 +1,81 @@
+import styled from 'styled-components'
+import WarningCircle from '../../../assets/svg-icons/warning-circle-icon.svg'
+
+interface StyleProps {
+  hasError: boolean
+}
+
+export const StyledWrapper = styled.div`
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-start;
+  flex-direction: column;
+  width: 100%;
+`
+
+export const Input = styled.input<StyleProps>`
+  width: 100%;
+  outline: none;
+  background-image: none;
+  background-color: ${(p) => p.hasError ? p.theme.color.errorBackground : p.theme.color.background02};
+  box-shadow: none;
+  border: ${(p) => p.hasError ? `4px solid ${p.theme.color.errorBorder}` : `1px solid ${p.theme.color.interactive08}`};
+  border-radius: 4px;
+  font-family: Poppins;
+  font-style: normal;
+  font-size: 13px;
+  line-height: 20px;
+  letter-spacing: 0.01em;
+  padding: 10px 10px 10px 10px;
+  color: ${(p) => p.theme.color.text01};
+  -webkit-box-shadow: none;
+  -moz-box-shadow: none;
```

Comment:
You can just use `box-shadow`. Actually I see you've already specified that property, so I don't think these are needed?

Reply:
You are correct :) will remove!